File: src/Python/easy/string/problem2744.py
from typing import List


class Solution:
    def maximumNumberOfStringPairs(self, words: List[str]) -> int:
        a = 0
        for i in range(0, len(words)):
            for j in range(i + 1, len(words)):
                if words[i] =="".join([words[j][1],words[j][0]]):
                    a += 1
        return a
    
    
    

words = ["cd","ac","dc","ca","zz"]

solution = Solution()
solution.maximumNumberOfStringPairs(words)

File: src/JavaScript/easy/string/problem2744.js
/**
 * @param {string[]} words
 * @return {number}
 */
var maximumNumberOfStringPairs = function(words) {
    let a = 0
    for(let i = 0; i < words.length; i++){
        for(let j = i + 1; j < words.length; j++){
            if(words[i] == [...words[j][1],words[j][0]].join("")){
                a++
            }
        }
    }
    return a
};


let words = ["cd","ac","dc","ca","zz"]

maximumNumberOfStringPairs(words)